doc/tutorials/access_store_ui.ipynb

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #44.            global sb_config, sb_store_file, sb_password
</pre><p>Do we need sb_store_file, sb_password as globals when sb_config is already global</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Yes, the %store doesn't work with local variables.</p>